src/sound/html5/HTML5AudioSoundManager.js

```diff
@@ -159,6 +159,21 @@ var HTML5AudioSoundManager = new Class({
         return sound;
     },
 
+    /**
+     * Sets the destination for the spatial sound.
+     * Currently only WebAudio is supported.
+     *
+     * @method Phaser.Sound.BaseSoundManager#setAudioDestination
+     * @override
+     * @since 3.6.0
+     *
+     * @param {Phaser.Types.Sound.SpatialSoundConfig|object} [destination] - An object with x and y fields
+     */
+    setAudioDestination: function ()
+    {
+
```

Comment:
There's no need for this method in here, because it's already in `BaseSoundManager`.